scanpipe/templates/scanpipe/resource_detail.html

```diff
@@ -111,17 +117,40 @@
         $el.addEventListener('click', function () {
           let active_li = document.querySelector("li.is-active");
           if (active_li) active_li.classList.remove("is-active");
+          prevItem.classList.remove("is-hidden");
+          nextItem.classList.remove("is-hidden");
           $el.parentElement.classList.add("is-active");
-
+          let count = 0;
```

Comment:
`count` is too generic, what are you counting?

Reply:
I have changed it to `valueIndex`